UnchainedLauncher.Core/Extensions/IEnumerableExtensions.cs

```diff
@@ -13,5 +15,14 @@ public static IEnumerable<T> Tap<T, T2>(this IEnumerable<T> source, Func<T, T2>
                 yield return item;
             }
         }
+
+        public static (IEnumerable<L>, IEnumerable<R>) SplitEithers<L, R>(this IEnumerable<Either<L, R>> el) {
```

Comment:
I think this method already exists as "Partition". https://github.com/Chiv2-Community/UnchainedLauncher/blob/main/UnchainedLauncher.Core/Services/Mods/Registry/GithubModRegistry.cs#L29 We use it here to convert a IEnumerable<Either<A, B>> in to a (IEnumerable<A>, IEnumerable<B>)